### DISCUSSION TITLE
Cannot confirm BlueSCSI SD card driver using SCSI Director Pro 4.0

### DISCUSSION THREAD

**Discussion by JDW1** [erichelgeson/BlueSCSI-v1#95] [General]
When I launch SCSI Director Pro 4.0 and choose View > Device Information... on my BlueSCSI drive, I see the following oddball text...

<img width="493" alt="image" src="https://user-images.githubusercontent.com/3928027/163717187-9daf8494-eb46-4bd7-9696-9be9d83bb361.png">

But when I check my Quantum 4GB spinner or check MacSD, I see that Apple's 7.3.5 driver was used:

<img width="495" alt="image" src="https://user-images.githubusercontent.com/3928027/163717233-3df36858-a609-4b86-8695-91fba479e30c.png">

**Comment by erichelgeson** [comment]
Thanks for the report @JDW1 - looks like an offset or something is wrong when reporting the info - we'll take a look!
